src/components/TokenLogo/tokenList.js

```diff
@@ -53,5 +54,10 @@ export const tokens = [
     name: 'FLASH',
     address: '0x5f0366c9962193fA774cdce9602195593B49f23C',
     logo: FlashLogo
+  },
+  {
+    name: 'OUTRACE',
```

Comment:
@kojiadrianojr  si name ba iyo ini su nag appear duman sa search? Baka dapat ORE ini instead of OUTRACE. Kung iyo, pa change nalang.